sympy/core/relational.py

```diff
@@ -23,6 +23,8 @@
 from sympy.multipledispatch import dispatch
 from .containers import Tuple
 from .symbol import Symbol
+from .add import Add
+from sympy.core.kind import NumberKind
```

Comment:
Does it not work to do `from .kind import` here?

Reply:
I was previously using VSCodium which seems to be a slower IDE compared to PyCharm at checking package dependencies. So I had written the import like that. I've changed the import to `from .kind import NumberKind` as you suggested.